openedx/core/djangoapps/content/search/api.py

```diff
@@ -62,6 +62,65 @@
 
 EXCLUDED_XBLOCK_TYPES = ['course', 'course_info']
 
+INDEX_DISTINCT_ATTRIBUTE = "usage_key"
```

Comment:
Nit: I like how you separated out these `INDEX_` settings, but I don't think they really belong in our public `api.py` file. What do you think about putting them in a new `index_config.py` module?